src/amo/components/AddonReviewCard/index.js

```diff
@@ -405,6 +402,43 @@ export class AddonReviewCardBase extends React.Component<InternalProps> {
         ],
       });
 
+      const byAuthorOnly = i18n.sprintf(i18n.gettext('by %(authorName)s'), {
+        authorName: review.userName,
+      });
+
+      const byAuthorLink = replaceStringsWithJSX({
+        text: i18n.sprintf(
+          i18n.gettext('by %(urlStart)s%(authorName)s%(urlEnd)s'),
+          {
+            authorName: review.userName,
+            urlStart: '%(urlStart)s',
+            urlEnd: '%(urlEnd)s',
+          },
+        ),
+        replacements: [
+          [
+            'urlStart',
+            'urlEnd',
+            (text) => (
+              <Link key={review.userName} to={`/user/${review.userId}`}>
+                <span>{text}</span>,{' '}
+              </Link>
+            ),
+          ],
+        ],
+      });
+
+      const byAuthor = !review.userUrl ? byAuthorOnly : byAuthorLink;
```

Comment:
The condition to render a link to the user profile is not about the `userUrl` in the `review` but whether the current user is an admin. Something like this: https://github.com/mozilla/addons-frontend/blob/48952905ceccb8e2209c76f063f8c649969fe388/src/amo/components/AddonAdminLinks/index.js#L159

Reply:
I had first created the condition with `userUrl` because I noticed that some `userUrl` were `null` in a handful of reviews. I thought this indicated that not all users had a profile page.

<img width="463" alt="Screen Shot 2020-03-30 at 4 02 02 PM" src="https://user-images.githubusercontent.com/28129806/77956439-df4db600-729f-11ea-9a8e-f6b91c369d65.png">

As advised, I used `hasAdminPermission` to check whether the current user is an admin.